plugins/woocommerce-blocks/assets/js/editor-components/switch-to-classic-shortcode-button/editor.scss

```diff
@@ -0,0 +1,31 @@
+.wc-blocks-switch-to-classic-shortcode-modal-content {
+	max-width: 480px;
+
+	p {
+		margin-top: 0;
+	}
+}
+
+.wc-blocks-switch-to-classic-shortcode-modal-actions {
+	text-align: right;
+
+	button {
+		margin-left: 4px;
+		display: inline-block;
+	}
+}
+
+ul.cross-list {
+	list-style: none outside;
+	margin: 0 0 24px 1em;
+	padding: 0;
+	li {
+		list-style: none outside;
+		margin: 1em 0;
+		padding: 0;
+	}
+	li::before {
+		content: "❌";
+		margin-right: 0.5em;
+	}
+}
```

Comment:
As we're using SCSS, we can change `li::before` to `&::before` and nest it within the `<li>` element. I'd also add line breaks. Here's my suggestion:

```suggestion
ul.cross-list {
  list-style: none outside;
  margin: 0 0 24px 1em;
  padding: 0;

  li {
    list-style: none outside;
    margin: 1em 0;
    padding: 0;

    &::before {
      content: "❌";
      margin-right: 0.5em;
    }
  }
}
```